airmozilla/popcorn/views.py

```diff
@@ -51,11 +42,72 @@ def event_meta_data(request):
 
     assert response.status_code == 302, response.status_code
 
+    return response.headers['location']
+
+
+def get_thumbnail(event):
+    image = event.picture and event.picture.file or event.placeholder_img
+    geometry = '160x90'
+    crop = 'center'
+
+    return thumbnail(image, geometry, crop=crop)
+
+
+@add_cors_header('*')
+@json_view
+def event_meta_data(request):
+    slug = request.GET.get('slug')
+    event = get_object_or_404(Event, slug=slug)
+    pattern = re.compile('(.+)\?')
+
+    base_url = get_video_url(event)
+    matches = pattern.match(base_url)
+    if matches:
+        video_url = matches.group()[:-1]
+    else:
+        video_url = base_url
+
+    thumb = get_thumbnail(event)
+
     return {
         'title': event.title,
         'description': event.short_description or event.description,
         'preview_img': thumb.url,
-        'video_url': response.headers['location'],
+        'video_url': video_url,
+    }
+
+
+@json_view
+@login_required
+def popcorn_data(request):
+    slug = request.GET.get('slug')
+    event = get_object_or_404(Event, slug=slug)
+
+    for edit in PopcornEdit.objects.filter(
+            event__slug=slug,
+            status=PopcornEdit.STATUS_SUCCESS).order_by('-created')[:1]:
+
+        data = edit.data
+        break
+    else:
+        video_url = get_video_url(event)
+        thumb = get_thumbnail(event)
+        data = {
+            "thumbnail": thumb.url,
+            "url": video_url,
+            "title": event.title,
+            "duration": event.duration,
+            "type": "AirMozilla",
```

Comment:
What is this `type`? If it's something that ever reaches users' eyes it should be `Air Mozilla`.